sdk/evaluation/azure-ai-evaluation/tests/unittests/test_evaluate.py

```diff
@@ -143,20 +143,25 @@ def test_evaluate_missing_required_inputs(self, missing_columns_jsonl_file):
         with pytest.raises(EvaluationException) as exc_info:
             evaluate(data=missing_columns_jsonl_file, evaluators={"g": F1ScoreEvaluator()})
 
-        assert "Missing required inputs for evaluator g : ['ground_truth']." in exc_info.value.args[0]
+        expected_message = "Some evaluators are missing required inputs:\n" "- g: ['ground_truth']\n"
+        assert expected_message in exc_info.value.args[0]
 
     def test_evaluate_missing_required_inputs_target(self, questions_wrong_file):
         with pytest.raises(EvaluationException) as exc_info:
             evaluate(data=questions_wrong_file, evaluators={"g": F1ScoreEvaluator()}, target=_target_fn)
-        assert "Missing required inputs for target : ['query']." in exc_info.value.args[0]
+        assert "Missing required inputs for target: ['query']." in exc_info.value.args[0]
 
-    def test_wrong_target(self, questions_file):
-        """Test error, when target function does not generate required column."""
+    def test_target_not_generate_required_columns(self, questions_file):
         with pytest.raises(EvaluationException) as exc_info:
-            # target_fn will generate the "response", but not ground truth.
+            # target_fn will generate the "response", but not "ground_truth".
             evaluate(data=questions_file, evaluators={"g": F1ScoreEvaluator()}, target=_target_fn)
 
-        assert "Missing required inputs for evaluator g : ['ground_truth']." in exc_info.value.args[0]
+        expected_message = "Some evaluators are missing required inputs:\n" "- g: ['ground_truth']\n"
+
+        expected_message2 = "Verify that the target function is generating the necessary columns for the evaluators."
```

Comment:
Would it be possible to print which columns were to be generated by target but target did not generate them ?

Reply:
That's a good idea. However, it's difficult to deduce the correct target columns, as the output from the target can be very dynamic. What we can do is print the columns generated by the target, so customers can quickly compare them with the evaluator's missing fields. That can help customers to identify the missing inputs root cause more quickly.

Reply:
This looks better?
![image](https://github.com/user-attachments/assets/1fa1d8c0-a772-4726-8595-ce01bada4683)